etna/transforms/math/sklearn.py

```diff
@@ -230,3 +244,13 @@ def _inverse_reshape(self, df: pd.DataFrame, transformed: np.ndarray) -> np.ndar
             [transformed[i * time_period_len : (i + 1) * time_period_len, :] for i in range(n_segments)], axis=1
         )
         return transformed
+
+    def get_regressors_info(self) -> List[str]:
+        """Return the list with regressors created by the transform."""
+        if self.in_column_regressor is None:
```

Comment:
Here you need to check whether self.out_column_regressors is not None and raise error that transform is not fitted if it is None